crates/astria-conductor/src/executor.rs

```diff
@@ -152,13 +152,13 @@ impl<C: ExecutionClient> Executor<C> {
                     self.alert_tx.send(Alert::BlockReceivedFromGossipNetwork {
                         block_height: block.header().height.value(),
                     })?;
-                    if let Err(e) = self
-                        .execute_block(SequencerBlockSubset::from_sequencer_block_data(
-                            *block,
-                            self.namespace,
-                        ))
-                        .await
-                    {
+                    let Some(block_subset) =
```

Comment:
Note the comment on `from_sequencer_block_data`. This change does not affect the execution of this block because `execute_block` checks again if the rollup data inside the block was empty (and we now mapped the case of "no rollup under the name, so use a default with no data" to `None`)